frontend/appflowy_flutter/lib/workspace/application/appearance.dart

```diff
@@ -15,6 +15,10 @@ import 'package:google_fonts/google_fonts.dart';
 part 'appearance.freezed.dart';
 
 const _white = Color(0xFFFFFFFF);
+const kDefaultFontFamily = 'Poppins';
```

Comment:
Can we create a class encapsulating these constants named: `DefaultAppearanceSettings`

Reply:
https://github.com/AppFlowy-IO/AppFlowy/pull/3137/commits/fc6516ef7c563f82e0f397a55c459c238e621b9c